smart-contracts/contracts/PublicLock.sol

```diff
@@ -53,6 +53,14 @@ contract PublicLock is ILockCore, ERC165, IERC721, IERC721Receiver, Ownable {
     uint refundPenaltyDenominator
   );
 
+  event Destroy(
+    address lock,
```

Comment:
I would not include the address of this contract in events (applies to both Destroy and Disable).  When reading events, the contract address is already known - making this redundant.

Reply:
done.